### PR TITLE
feat: remove calendar top bar

### PR DESCRIPTION
**Pull request by DarkPhoenix2704**
## Change Summary

Provide summary of changes with issue number if any.

Update the calendar style. 
Removed the Calendar topbar and moved them to toolbar and side menu
Updated posthog events
Updated tests to new ui

## Change type

- [x] feat: (new feature for the user, not a new feature for build script)
- [x] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [x] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

File: packages/nc-gui/components/smartsheet/calendar/index.vue [hunk 1]
```diff
@@ -105,6 +98,7 @@ const expandRecord = (row: RowType, state?: Record<string, any>) => {
 const newRecord = (row: RowType) => {
   // TODO: The default values has to be filled based on the active calendar view
   // and selected sidebar filter option
+  $e('c:calendar:new-record', activeCalendarView.value)
```
[the remark this answers is not in the document]
Comment (DarkPhoenix2704) [reply packages/nc-gui/components/smartsheet/calendar/index.vue:101]: ignore

[the remark this answers is not in the document]
Comment (DarkPhoenix2704) [reply packages/nc-gui/components/smartsheet/calendar/index.vue:101]: ignore

File: packages/nc-gui/components/smartsheet/calendar/DayView/DateTimeField.vue [hunk 2]
```diff
@@ -804,6 +808,7 @@ const dropEvent = (event: DragEvent) => {
       dragElement.value = null
     }
     updateRowProperty(newRow, updateProperty, false)
+    $e('c:calendar:day:drag-record')
```
[the remark this answers is not in the document]
Comment (DarkPhoenix2704) [reply packages/nc-gui/components/smartsheet/calendar/DayView/DateTimeField.vue:811]: they are posthog events. ignore

**Comment by o1lab** [comment]
- [ ] week - only month and year
- [ ] year : fix symmetry for day header
- [ ] day / week : we need pointer instead of cursor
- [ ] year : we need double click to come to month
- [ ] there is a input box in settings - may be a bug : image

- [ ] also can we show the full field name there is enough width

**Comment by o1lab** [comment]
- [ ] Take out the line between day header and dates
image

- [ ] Make the weekend dates grey
- [ ] Lets just one of the next, previous buttons. And remove the other.

**Review by dstala — APPROVED** [review]

**Pull request merged**